ports/getdns/portfile.cmake

```
@@ -0,0 +1,44 @@
+set(GETDNS_VERSION 1.6.0)
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "uwp")

set(GETDNS_VERSION 1.6.0)
```
Since it currently doesn't support uwp, it would be better to add the messages here.

---

Argh! Sorry. I'd thought that a CONTROL entry was sufficient.